### PR TITLE
Move imports outside of test method code

### LINKED ISSUE #266: Remove imports from within test functions
PEP8 specifies that imports should be put at the top of a Python file: https://peps.python.org/pep-0008/#imports

Therefore, linters will complain for the tests explained in [Episode 2, Section 2, "Scaling Up Unit Testing"](https://carpentries-incubator.github.io/python-intermediate-development/22-scaling-up-unit-testing/index.html).

We should group all imports at the top of the test files unless we want this to be a teachable linting moment.

### PR DESCRIPTION
**Pull request by johnnonweiler**
Closes #266

### CODE REVIEW

**Review by bielsnohr — APPROVED** [review]
Quite a simple change. LGTM.

File: _episodes/22-scaling-up-unit-testing.md [hunk 1]
```diff
@@ -85,6 +86,8 @@ and adding more tests scales better as our code becomes more complex.
 > 
 > > ## Solution
 > > ~~~
+> > from inflammation.models import daily_max
+> > from inflammation.models import daily_min
```
Comment (bielsnohr) [inline _episodes/22-scaling-up-unit-testing.md:90]: We can make this into a single line
```suggestion
> > from inflammation.models import daily_max, daily_min
```

**Review by bielsnohr — CHANGES_REQUESTED** [review]
On second inspection, there are still some cases in the content where import statements are within code snippets, namely `21-automatically-testing-software.md`.

@johnnonweiler if you give me permission to add commits [like this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) then I can add the subsequent changes needed.

**Comment by bielsnohr** [comment]
> On second inspection, there are still some cases in the content where import statements are within code snippets, namely `21-automatically-testing-software.md`.
> 
> @johnnonweiler if you give me permission to add commits [like this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) then I can add the subsequent changes needed.

Ah, I just realised that I can actually make edits to this PR directly, so no need to do the step in the link I sent.

**Comment by bielsnohr** [comment]
Okay, I think I did a good enough regex search to get all the instances of modules being imported inside of a test function. Because this change expanded beyond what I initially thought it would be, we likely need to verify that all or most of the code snippet changed actually still run properly. I don't have the time for that at the moment, however, we will be running the course again at UKAEA, so we could be the test subjects.

**Comment by steve-crouch** [comment]
Looks good, we'll just need to verify these changes then. After discussion with Aleks, due to time constraints, retiring this PR for now until the material is transitioned to the new Carpentries format, when we'll recreate it and can work on it from there.

**Review by bielsnohr — APPROVED** [review]
I have found some time to go through most of the code snippet changes and incorporate them into the starting point that https://github.com/carpentries-incubator/python-intermediate-inflammation/pull/18 will implement. Everything works fine, so I am going to approve this.

**Pull request merged**